examples/multisig_transfer.ts

```diff
@@ -0,0 +1,121 @@
+import { createKeyMulti } from "../deps/polkadot/util_crypto.ts";
+import { SendAndWatchExtrinsicProps } from "../effect/mod.ts";
+import * as C from "../mod.ts";
+import * as t from "../test_util/mod.ts";
+import * as U from "../util/mod.ts";
+
+type KeyringPair = typeof t.alice;
+
+const config = await t.config();
+const SIGNATORIES: Uint8Array[] = [
+  t.alice.publicKey,
+  t.bob.publicKey,
+  t.charlie.publicKey,
+];
+const THRESHOLD = 2;
+const ABC_MULTISIG_ADDRESS = createKeyMulti(SIGNATORIES, THRESHOLD);
+
+const addBalanceToMultisigAddress = () =>
+  C.sendAndWatchExtrinsic({
+    config,
+    sender: {
+      type: "Id",
+      value: t.alice.publicKey,
+    },
+    palletName: "Balances",
+    methodName: "transfer_keep_alive",
+    args: {
+      value: 10_000_000_000_000n,
+      dest: {
+        type: "Id",
+        value: ABC_MULTISIG_ADDRESS,
+      },
+    },
+    sign(message) {
+      return {
+        type: "Sr25519",
+        value: t.alice.sign(message),
+      };
+    },
+    createWatchHandler,
+  });
+
+interface CreateOrApproveProposalProps {
+  sender: KeyringPair;
+  maybe_timepoint?: Timepoint;
+}
+
+interface Timepoint {
+  height: number;
+  index: number;
+}
+
+const createOrApproveProposal = ({ sender, maybe_timepoint }: CreateOrApproveProposalProps) =>
+  C.sendAndWatchExtrinsic({
+    config,
+    sender: {
+      type: "Id",
+      value: sender.publicKey,
+    },
+    palletName: "Multisig",
+    methodName: "as_multi",
+    args: {
+      threshold: THRESHOLD,
+      call: {
+        type: "Balances",
+        value: {
+          type: "transfer",
+          dest: {
+            type: "Id",
+            value: t.dave.publicKey,
+          },
+          value: 1_000_000_000_000n,
+        },
+      },
+      other_signatories: SIGNATORIES
+        .filter((value) => value !== sender.publicKey),
+      store_call: false,
+      max_weight: 0n,
+      maybe_timepoint,
+    },
+    sign(message) {
+      return {
+        type: "Sr25519",
+        value: sender.sign(message),
+      };
+    },
+    createWatchHandler,
+  });
+
+const _getTimepoint = () => {
+  return C.readEntry(config, "Multisig", "Multisigs", [
+    ABC_MULTISIG_ADDRESS,
+  ]);
```

Comment:
The timepoint is required for any approval after the proposal creation

<img width="1482" alt="image" src="https://user-images.githubusercontent.com/1209171/195434238-2bb4b83d-62b7-402a-a60d-3ed9eba3aba2.png">

Reply:
Depends on https://github.com/paritytech/capi/issues/294